app/controllers/patients_controller.rb

```diff
@@ -314,6 +314,19 @@ def update_status
     patient = current_user.get_patient(params.permit(:id)[:id])
     redirect_to(root_url) && return if patient.nil?
 
+    # update LDE for patient and group members only in the exposure workflow with continuous exposure on (separate from updating monitoring status)
+    if params.permit(:apply_to_group_cm_exp_only)[:apply_to_group_cm_exp_only]
+      ([patient] + (current_user.get_patient(patient.responder_id)&.dependents&.where(continuous_exposure: true, isolation: false) || [])).uniq.each do |member|
+        # turn off continuous exposure if LDE is updated
```

Comment:
might be worth adding `next unless params[:apply_to_group_cm_exp_only_date].present?` for validation similar to the block of code before

Reply:
done